.travis.yml

```diff
@@ -316,6 +316,7 @@ script:
 
   # make sure CXX is correctly set
   - if [[ "${COMPILER}" != "" ]]; then export CXX=${COMPILER}; fi
+  - if [[ "${CXXFLAGS}" != "" ]]; then export CXXFLAGS=${CXXFLAGS}; fi
```

Comment:
This line has no effect. It is probably intended to activate C++17 (solved by #2540).
```suggestion
```

Reply:
Fixed in 384442e8e5c7b684e2b1cc4acf0047d4d38e66d5